website/docs/onboarding/containers/installing-podman.md

```diff
@@ -0,0 +1,60 @@
+---
+sidebar_position: 10
+title: Installing Podman
+description: Podman Desktop can assist you to install Podman on Windows and macOS.
+tags: [podman-desktop, podman, installing, windows]
+keywords: [podman desktop, containers, podman, installing, installation, windows, macos]
+---
+
+# Installing Podman with Podman Desktop
+
+On Windows and macOS, running the Podman container engine requires running a Linux distribution on a virtual machine.
+Podman Desktop can assist you to install the Podman container engine in a Fedora distribution of Linux, on a virtual machine.
+
+Main benefits are:
+
+- Ease of use.
+- On Windows: Windows Subsystem for Linux version 2 (WSL 2) native virtualization performance.
+
+Consider rather [installing Podman with OpenShift Local](/docs/onboarding/containers/installing-podman-with-openshift-local-on-windows) if your environment doesn't allow you to meet the prerequisites.
+
+#### Prerequisites
+
+- Podman is not installed.
+- 6 GB RAM.
+- The host runs Windows or macOS.
+- Windows prerequisites:
+  - [WSL prerequisites](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+    - User with administrator privileges.
+    - Windows 64bit.
+    - Windows 10 Build 18362 or greater, or Windows 11.
```

Comment:
The build version must be updated to 19043 -> https://github.com/containers/podman-desktop/pull/4255

Reply:
Updated.